doc/module-docs/cc_ubuntu_autoinstall/data.yaml

```diff
@@ -0,0 +1,18 @@
+cc_ubuntu_autoinstall:
+  description: |
+    The `autoinstall` key indicates a configuration for the Ubuntu installer.
+    It is not acted on by cloud-init other than to ensure that the
+    configuration is schema compliant and that the installer package is
+    present on the system.
+
+    .. note::
+      The installer may use the provided configuration to instrument
+      cloud-init on the target system. See
+      `the Ubuntu installer documentation <https://canonical-subiquity.readthedocs-hosted.com/en/latest/reference/autoinstall-reference.html#ai-user-data>`_
```

Comment:
Inspecting the source of that autoinstall page I see a better anchor (complete with a title that says link to this id)
`<a class="headerlink" href="#user-data" title="Link to this heading">¶</a>`
```suggestion
      `the Ubuntu installer documentation <https://canonical-subiquity.readthedocs-hosted.com/en/latest/reference/autoinstall-reference.html#user-data>`_
```